polaris-migrator/src/migrations/scss-remove-unused-import/scss-remove-unused-import.ts

```diff
@@ -0,0 +1,58 @@
+import type {FileInfo, API, Options} from 'jscodeshift';
+import postcss from 'postcss';
+import type {Plugin} from 'postcss';
+
+import {createInlineComment} from '../../utilities/sass';
+import {POLARIS_MIGRATOR_COMMENT} from '../../constants';
+
+interface PluginOptions extends Options {
+  importPath?: string;
```

Comment:
I believe the Sass module system refers to these as URLs:
https://github.com/Shopify/polaris/pull/7322/files#diff-2d9c77c4c87bf51828a048bdd0a559a0657560a0e7656eaa80932079afe3f68dR43